src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -232,12 +228,18 @@ export function MultiSelect<T>({
      * Update selected items if we get new selected items from outside
      */
     React.useEffect(() => {
-        if (!externalSelectedItems) {
+        // An infinite loop can occur when both prePopulateWithItems and selectedItems props are provided.
```

Comment:
@haschek I am not 100% sure that it's ok to set both `prePopulateWithItems` and outer `selectedItems` properties together. We may just throw an error when both properties are provided, in this case we won't need to use `initialized` ref.

Reply:
Decided to separate the usage of 2 props.